client/pages/home.js

```diff
@@ -35,8 +35,15 @@ module.exports = function (state, emit) {
             A distributed <span class="color-green">dat</span>a community
           </h1>
           <h2 class="f3 mt3 mb3 color-neutral-50 fw2">
-            Dat is a nonprofit-backed community & open protocol for building apps of the future.
+            Dat is a nonprofit-backed data sharing protocol for applications of the future.
           </h2>
+
+          ${cta(state, emit, {
+            href: 'https://docs.datproject.org',
```

Comment:
maybe this could point to the install page?

Reply:
ya changed to `install now` and pointed to install. I really need to make that install page look nice like the old one!